prime-dotnet-webapi/Models/EnrolmentCertificate.cs

```diff
@@ -35,9 +36,17 @@ public static EnrolmentCertificate Create(Enrollee enrollee)
                 PreferredLastName = enrollee.PreferredLastName,
                 DateOfBirth = enrollee.DateOfBirth,
                 LicensePlate = enrollee.LicensePlate,
-                Privileges = enrollee.AssignedPrivileges.Select(ap => ap.Privilege),
                 OrganizationTypes = enrollee.Organizations.Select(org => org.OrganizationType),
-                EnrolmentCertificateNote = enrollee.EnrolmentCertificateNote
+                EnrolmentCertificateNote = enrollee.EnrolmentCertificateNote,
+                Transactions = enrollee.AssignedPrivileges
+                                .Select(ap => ap.Privilege)
+                                .Where(p => p.PrivilegeGroup.PrivilegeTypeCode == PrivilegeType.PHARMANET_TRANSACTIONS_TYPE),
+                UserType = enrollee.AssignedPrivileges
+                            .Select(ap => ap.Privilege)
+                            .Where(p => p.PrivilegeGroupCode == PrivilegeGroup.USER_TYPE_GROUP).SingleOrDefault(),
+                CanHaveOBOs = enrollee.AssignedPrivileges
+                            .Select(ap => ap.Privilege)
+                            .Where(p => p.PrivilegeGroupCode == PrivilegeGroup.CAN_HAVE_OBOS_GROUP).SingleOrDefault(),
```

Comment:
A little nit-picky, but might as well drop the _TYPE and _GROUP from these constants. It's a little redundant.